concepts/05 UI Components/Switch/00 Getting Started with Switch/15 Configure Size and Representation.md

```diff
@@ -0,0 +1,56 @@
+The component allows you to configure its size and representation. To specify a width, assign the needed value to the [width](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#width) property. Use the [rtlEnabled](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#rtlEnabled) property to switch the widget to a right-to-left representation. If you set this property to `true`, the Switch label flows from right to left, and the layout of elements is reversed.
+
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#switch").dxSwitch({
+            // ...
+            width: 80,
+            rtlEnabled: true,
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-switch ...
+        [width]="80"
+        [rtlEnabled]="true"
+    >
+    </dx-switch>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxSwitch ...
+            :width="80"
+            :rtlEnabled="true"
+        />
+    </template>
+
+    <script>
+    // ...
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    // ...
+    
+    function App() {
+        // ...
+        return (
+            <Switch ...
+                width="80"
```

Comment:
Not a big deal but just for consistency better use `width={80}` here, because on others tabs width is set as number